### DISCUSSION TITLE
configure script: packages with dashes (-) will create invalid PHP code in README usage section

### DISCUSSION THREAD

**Discussion by levu42** [spatie/package-skeleton-php#48] [Q&A]
Hello! If I create a package with a - in the package name, the usage section in the README uses the dashes directly for a variable name, resulting in incorrect PHP.

Example: Running the configure script with vendor = Me, package name = my-first-package, class name = MFP results in:

\#\# Usage
\`\`\`php
$my-first-package = new Me\MPF();

when it should convert the variable name e.g. with slugify to $myfirstpackage.

Thanks for the great video course & this helpful skeleton!

**Comment by freekmurze** [comment]
Feel free to submit a PR that fixes this.

Thanks for the kind words on our work 👍
